js/index.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
```

Comment:
this should be tested when I did it for BRP it was rushed out. I think we should do it properly now